concent_api/middleman/constants.py

```diff
@@ -9,6 +9,7 @@
 STANDARD_ERROR_MESSAGE = "This connection has been already added"
 WRONG_TYPE_ERROR_MESSAGE = "QueuePool can only contain mapping between connection ID (int) and response queue (asyncio.Queue)"
 AUTHENTICATION_CHALLENGE_SIZE = 128
+HEARTBEAT_REQUEST_ID = 777
```

Comment:
Isn't now there a possibility that the 'regular' request will be sent with that id too?

Reply:
There is, but his id isn't really taken into account (`Paylodtype` matters)during the communication with SS. Nevertheless, I came to conclusion that it would be better to have a constant `request_id` for a `HeartbeatFrame`